qiskit/transpiler/coupling.py

```diff
@@ -186,6 +190,27 @@ def shortest_undirected_path(self, physical_qubit1, physical_qubit2):
             raise CouplingError(
                 "Nodes %s and %s are not connected" % (str(physical_qubit1), str(physical_qubit2)))
 
+    @property
+    def is_symmetric(self):
+        """
+        Test if the graph is symmetric.
+
+        Return True if symmetric, False otherwise
+        """
+        if self._is_symmetric is None:
+            self._is_symmetric = self._check_symmetry()
+        return self._is_symmetric
+
+    def _check_symmetry(self):
+        """
+        Calculates symmetry
+
+        Returns:
+            Bool: True if symmetric, False otherwise
+        """
+        mat = nx.adjacency_matrix(self.graph)
+        return (abs(mat - mat.T) > 1e-10).nnz == 0
```

Comment:
Is the `1e-10` check required? Edges in `mat` should only be integers (the number of edges between two nodes).

Reply:
Fixed in 407723d2